common/config/presets/windows.toml

```diff
@@ -215,7 +215,7 @@ tor_control_auth = "none" # or "password=xxxxxx"
 # The onion port to use.
 tor_onion_port = 18150
 # The address to which traffic on the node's onion address will be forwarded
-tor_forward_address = "/ip4/127.0.0.1/tcp/18150"
+# tor_forward_address = "/ip4/127.0.0.1/tcp/0"
```

Comment:
Please change line 304 as well:
`#tor_forward_address = "/ip4/127.0.0.1/tcp/18141"` -> `#tor_forward_address = "/ip4/127.0.0.1/tcp/0"`

Reply:
Didn't see that one - thanks - done